website/templates/project/settings.mako

```diff
@@ -531,6 +462,74 @@
 
         % endif  ## End Retract Registration
 
+        % if enable_institutions:
+             <div class="panel panel-default scripted" id="institutionSettings">
+                 <span id="configureInstitutionAnchor" class="anchor"></span>
+                 <div class="panel-heading clearfix">
+                     <h3 class="panel-title">Project Affiliation / Branding</h3>
+                 </div>
+                 <div class="panel-body">
+                     <div class="help-block">
+                         % if 'write' not in user['permissions']:
+                             <p class="text-muted">Contributors with read-only permissions to this project cannot add or remove institutional affiliations.</p>
+                         % endif:
+                         <!-- ko if: affiliatedInstitutions().length == 0 -->
+                         Projects can be affiliated with institutions that have created OSF for Institutions accounts.
+                         This allows:
+                         <ul>
+                            <li>institutional logos to be displayed on public projects</li>
+                            <li>public projects to be discoverable on specific institutional landing pages</li>
+                            <li>single sign-on to the OSF with institutional credentials</li>
+                            <li><a href="http://help.osf.io/m/os4i">FAQ</a></li>
+                         </ul>
+                         <!-- /ko -->
+                     </div>
+                     <!-- ko if: affiliatedInstitutions().length > 0 -->
+                     <label>Affiliated Institutions: </label>
+                     <!-- /ko -->
+                     <table class="table">
+                         <tbody>
+                             <!-- ko foreach: {data: affiliatedInstitutions, as: 'item'} -->
+                             <tr>
+                                 <td><img class="img-circle" width="50px" height="50px" data-bind="attr: {src: item.logo_path}"></td>
+                                 <td><span data-bind="text: item.name"></span></td>
+                                 <td>
+                                     % if 'admin' in user['permissions']:
+                                         <button data-bind="disable: $parent.loading(), click: $parent.clearInst" class="pull-right btn btn-danger">Remove</button>
+                                     % elif 'write' in user['permissions']:
+                                         <!-- ko if: $parent.userInstitutionsIds.indexOf(item.id) !== -1 -->
+                                            <button data-bind="disable: $parent.loading(), click: $parent.clearInst" class="pull-right btn btn-danger">Remove</button>
```

Comment:
Question I don't know the answer to: do all buttons need `aria-label`'s or just those whose content don't make it clear what they do?

My guess is that it cannot hurt and it could be even more explicit that way.

Reply:
@Alex, I do not believe `aria-label`s are needed because the buttons have text in this case.